assembl/static2/js/app/components/administration/discussion/preferences/index.jsx

```diff
@@ -8,10 +8,10 @@ import { Field, FormSpy } from 'react-final-form';
 import classnames from 'classnames';
 import Loader from '../../../common/loader';
 import LoadSaveReinitializeForm from '../../../form/LoadSaveReinitializeForm';
-import AdminForm from '../../../../components/form/adminForm';
-import CheckboxFieldAdapter from '../../../../components/form/checkboxFieldAdapter';
+import AdminForm from '../../../form/adminForm';
```

Comment:
were the components moved ?

Reply:
It's the same path without the unnecessary ../component 
It's a new rule that reformat the path to the easiest path